src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/ProcessController.cs

```diff
@@ -96,6 +96,11 @@ public class ProcessController : ControllerBase
 
             string altinnTaskType = existingInstance.Process?.CurrentTask?.AltinnTaskType;
 
+            if (processState != null && processState.CurrentTask != null & processState.CurrentTask.FlowType != null && !processState.CurrentTask.FlowType.Equals("CompleteCurrentMoveToNext"))
```

Comment:
This can be simplified 
```suggestion
            if (processState?.CurrentTask?.FlowType != null && !processState.CurrentTask.FlowType.Equals("CompleteCurrentMoveToNext"))
```